docs/pages/components/Notice.mdx

```diff
@@ -112,6 +112,48 @@ function DismissableNotice() {
 }
 ```
 
+## Variants
+
+The Notice component now includes a variant prop that allows for two display styles: default and condensed.
+
```

Comment:
I think some of the existing documentation needs to be re-worked a bit. We should update to specify that the info/caution/danger properties are types and not variants.